frontend/javascripts/oxalis/view/node_context_menu.js

```diff
@@ -226,9 +227,14 @@ function NoNodeContextMenuOptions({
       className="node-context-menu-item"
       key="load-mesh-file"
       onClick={loadMesh}
-      disabled={!currentMeshFile}
+      disabled={!currentMeshFile || !features().jobsEnabled}
     >
-      Load Precomputed Mesh
+      <Tooltip
+        visible={!features().jobsEnabled}
+        title="The computation of mesh files is not supported by this webKnossos instance."
+      >
```

Comment:
loading a mesh file should work even if jobs are disabled (only computation of meshes is bound to jobs), since one can manually put a mesh file on disk even without jobs.